ansible/configs/sap-smart/pre_software.yml

```diff
@@ -0,0 +1,86 @@
+- name: Configure RHEL 8 hosts with Repositories 
+  hosts:
+    - bastions:towers
```

Comment:
For consistency in formatting across the config, can we keep the hosts list on one line for all of these plays?

Reply:
Yes, I did not notice that. Changed all that apply